dart/lib/src/sentry_options.dart

```diff
@@ -139,7 +139,8 @@ class SentryOptions {
 
   /// Sentry client name used for the HTTP authHeader and userAgent eg
   /// sentry.{language}.{platform}/{version} eg sentry.java.android/2.0.0 would be a valid case
-  String? sentryClientName;
+  String get sentryClientName =>
+      '${sdkName(platformChecker.isWeb)}/$sdkVersion';
```

Comment:
Can we read from the `sdk` directly? so we don't need to call the function again.

Reply:
Good catch! 👍